Cargo.toml

```diff
@@ -3,3 +3,15 @@ members = [
 	"core",
 	"registry-resolver"
 ]
+[package]
+name = "ssi"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Comment:
Why are we adding dependencies here? This is the top level `Cargo.toml` we should add the dependencies for the core functionality in the `core/Cargo.toml` file